File: php/car.php
<?php
require_once('account.php');
class Car {
    public $id;
    public $license;
    public $driver;
    protected $passenger;

    public function __construct($license, $driver) {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar() {
        echo "my-license: $this->license
                my-driver: {$this->driver->name}
                    total-passengers: $this.passenger
                    ";
        }

    public function getPassenger() {
        return $this->passenger;
}
    public function setPassenger($passenger) {
            if ($passenger == 4) {
                $this->passenger = $passenger;
                }
                else {
                echo "You need to assign 4 passengers";
                }
            }
}
?>

File: java/car.java
class Car {
    Integer id;
    String license;
    Account driver;
    private Integer passenger;

    public Car(String license, Account drive) {
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        if ( passenger != null) {
        System.out.println("License:" + license + "Driver" + driver + "Passenger" + passenger);
        }
    }

    public Integer getPassenger() {
        return passenger;
    }

    public void setPassenger(Integer passenger) {
        if(passenger == 4) {
                this.passenger = passenger;
        } else {
                System.out.println("Necesitas asignar 4 pasajeros");
        }
    }
}